README.md

```diff
@@ -1 +1,234 @@
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Understood the concept of version control.**
+- **Write short, detailed, easy to understand descriptions of pull requests.**
+- **Professional documentation of Readme**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+NPM installed
+Any code editor
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder by:
+cd my-folder
+git clone https://github.com/mershark/Hello-microverse-project.git
+cd my-project
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Installation is completed once the repository and cloned into your local machine.
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project launch index.html in your browser
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+npx stylelint "**/*.{css,scss}
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project by transferring the files in your local repository onto your sever
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mershark Takyi**
+
+- GitHub: [@githubhandle](https://github.com/mershark)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mershark/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Home page**
+- [ ] **About us page**
+- [ ] **Blog Page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, your support will be very much appreciate, kindly contact me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my code reviewer who gave me insightful reviews to help correct and complete this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Ouch! this is so close but it seems like your MIT link is broken as it's showing a "404 page not found" error and this will make your project look unprofessional so I highly advise you fix it.
- [ ] So the problem is from the `(./LICENSE)` in line `230` that text should be replaced with `./MIT.md` so It can properly link to the `MIT.md` file you have in your root directory. 😉 _This is just a suggestion please make sure it works before submitting for a review as there might be other problems_

Reply:
Thanks for the review @AbelG101